python/assistant-stream/src/assistant_stream/create_run.py

```
@@ -74,13 +83,16 @@ async def create_run(
     async def background_task():
         try:
             await callback(controller)
-
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)